charts/trino/templates/secret-extra.yaml

```diff
@@ -0,0 +1,14 @@
+{{- range .Values.secrets }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: "{{ .name }}"
```

Comment:
Better to use a template variable. We may use `trino.catalog` (the current name of config map) or use `trino.catalog-secret` to avoid conflicts.